### PR TITLE
Launcher multinode

### PR DESCRIPTION
**Pull request by al1img**

### CODE REVIEW

File: launcher/launcher.go [hunk 1]
```diff
@@ -442,14 +303,14 @@ func (launcher *Launcher) RestartInstances() error {
 	launcher.Lock()
```
Comment (Eabramov84) [inline launcher/launcher.go:304]: RestartInstances will contains list of instances.

File: launcher/launcher.go [hunk 2]
```diff
@@ -627,110 +394,77 @@ func (launcher *Launcher) runInstances(runInstances []InstanceInfo) {
...
 
-				if err := launcher.serviceProvider.ApplyService(service.ServiceInfo); err != nil {
-					log.WithField("serviceID", service.ServiceID).Errorf("Can't apply service: %v", err)
+	currentInstances := make([]*runtimeInstanceInfo, 0, len(launcher.currentInstances))
 
-					launcher.errorServices = append(launcher.errorServices,
-						service.cloudStatus(cloudprotocol.ErrorStatus, err))
-				}
+	for _, currentInstance := range launcher.currentInstances {
+		currentInstances = append(currentInstances, currentInstance)
```
Comment (MykolaSolyanko) [inline launcher/launcher.go:418]: you could use `[]` instead `append`

File: launcher/service.go [hunk 3]
```diff
@@ -72,13 +74,6 @@ func (launcher *Launcher) cacheCurrentServices(instances []InstanceInfo) {
 		}
 
 		launcher.currentServices[instance.ServiceID] = &service
-
-		if service.err == nil {
```
Comment (MykolaSolyanko) [inline launcher/service.go:76]: why did you remove UseService, in this function the timestamp of the service is set, which affects the TTL of the removal of the service and the layer ?

File: launcher/launcher.go (hunk 1 above)
Comment (al1img) [reply launcher/launcher.go:304]: But why? `RestartInstances` is used when unitconfig is updated to restart already started instances. What is the point to use RestartInstances with list of instances?

File: launcher/launcher.go (hunk 2 above)
Comment (al1img) [reply launcher/launcher.go:418]: I can't use it directly. launcher.currentInstances is the map. 
```go
i := 0

for _, currentInstance := range launcher.currentInstances {
    currentInstances[i] = currentInstance
    i++ 
}
```

with append it is shorter.

File: launcher/instance.go [hunk 4]
```diff
@@ -21,6 +21,7 @@ import (
 	"encoding/hex"
 	"path/filepath"
 
+	"github.com/aoscloud/aos_common/aostypes"
 	"github.com/aoscloud/aos_common/api/cloudprotocol"
 	"github.com/aoscloud/aos_servicemanager/runner"
```
Comment (Eabramov84) [inline launcher/instance.go:25]: move github.com/aoscloud/aos_servicemanager/runner" to local import

File: launcher/instance.go [hunk 5]
```diff
@@ -21,6 +21,7 @@ import (
 	"encoding/hex"
 	"path/filepath"
 
+	"github.com/aoscloud/aos_common/aostypes"
```
Comment (Eabramov84) [inline launcher/instance.go:23]: move github.com/aoscloud/aos_servicemanager/runner" to local import

File: launcher/launcher.go (hunk 1 above)
Comment (Eabramov84) [reply launcher/launcher.go:292]: After unit config changes rebalanicig will be performed and list of instances could be changed

**Review by MykolaSolyanko — APPROVED** [review]

**Review by Eabramov84 — APPROVED** [review]
